src/renderer/services/wallet/balances.ts

```diff
@@ -139,7 +140,7 @@ const btcLedgerBalance$ = FP.pipe(
 /**
  * Transforms ETH data (address + `WalletBalance`) into `ChainBalance`
  */
-const ethChainBalance$: ChainBalance$ = Rx.combineLatest([ETH.address$, ETH.balances$]).pipe(
+const ethChainBalance$: ChainBalance$ = Rx.combineLatest([ETH.address$, ETH.balances$(ERC20Assets)]).pipe(
```

Comment:
`ETHAssets` (see previous comment).


Pls add a note this list is for testnet only.